homeassistant/components/climate/wink.py

```diff
@@ -18,15 +18,13 @@
 from homeassistant.const import (
     TEMP_CELSIUS, STATE_ON,
     STATE_OFF, STATE_UNKNOWN)
+from homeassistant.components.fan import (SPEED_LOW, SPEED_MEDIUM,
```

Comment:
I don't think we should import from the fan component. Put them in const.py or redefine them in climate component.